docs/resources/control-flow/backend-logic/backend-query/query-collection.md

```diff
@@ -80,6 +80,16 @@ The instructions to query a Supabase table are almost the same, except that for
     </iframe>
 </div>
 
+:::warning[Limitations of Supabase Streaming with Filters]
+When using Supabase query with real-time updates enabled, you have the following limitations:
+
+1. **Only One Filter is Supported:** Supabase streaming supports only a single filter. Combining multiple filters (e.g., `isActive = true AND city = 'Los Angeles'`) is not allowed.
```

Comment:
Should be a bullet list instead of ordered since this is not a ordered list right?

Reply:
Yes, updated.